web/cypress/fixtures/hfi-calc/stations.json

```diff
@@ -0,0 +1 @@
ne/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e3b8-0210-e053-259e228e58c7"},"planning_area":{"name":"Vernon (K4)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":388,"station_props":{"name":"KETTLE 2","elevation":1389,"fuel_type":{"abbrev":"C5","description":"Red and white pine"},"wfwx_station_uuid":"bfe0a6e2-e412-0210-e053-259e228e58c7"},"planning_area":{"name":"Vernon (K4)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":328,"station_props":{"name":"PENTICTON RS","elevation":427,"fuel_type":{"abbrev":"C7","description":"Ponderosa pine/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e3d6-0210-e053-259e228e58c7"},"planning_area":{"name":"Penticton (K5)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":334,"station_props":{"name":"MCCUDDY","elevation":1067,"fuel_type":{"abbrev":"C7","description":"Ponderosa pine/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e3dc-0210-e053-259e228e58c7"},"planning_area":{"name":"Penticton (K5)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":836,"station_props":{"name":"AUGUST LAKE","elevation":855,"fuel_type":{"abbrev":"C7","description":"Ponderosa pine/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e2df-0210-e053-259e228e58c7"},"planning_area":{"name":"Merritt (K6)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":1399,"station_props":{"name":"MERRITT 2 HUB","elevation":640,"fuel_type":{"abbrev":"C7","description":"Ponderosa pine/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e23c-0210-e053-259e228e58c7"},"planning_area":{"name":"Merritt (K6)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":280,"station_props":{"name":"LILLOOET","elevation":408,"fuel_type":{"abbrev":"C7","description":"Ponderosa pine/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e2ae-0210-e053-259e228e58c7"},"planning_area":{"name":"Lillooet (K7)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":1055,"station_props":{"name":"SPLINTLUM","elevation":424,"fuel_type":{"abbrev":"C7","description":"Ponderosa pine/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e14c-0210-e053-259e228e58c7"},"planning_area":{"name":"Lillooet (K7)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":1029,"station_props":{"name":"FIVE MILE","elevation":865,"fuel_type":{"abbrev":"C7","description":"Ponderosa pine/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e137-0210-e053-259e228e58c7"},"planning_area":{"name":"Lillooet (K7)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":309,"station_props":{"name":"GWYNETH LAKE","elevation":1205,"fuel_type":{"abbrev":"C7","description":"Ponderosa pine/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e3c3-0210-e053-259e228e58c7"},"planning_area":{"name":"Lillooet (K7)","fire_centre":{"name":"Kamloops Fire Centre"}}},{"code":306,"station_props":{"name":"FRENCH BAR","elevation":1320,"fuel_type":{"abbrev":"C7","description":"Ponderosa pine/Douglas fir"},"wfwx_station_uuid":"bfe0a6e2-e3c0-0210-e053-259e228e58c7"},"planning_area":{"name":"Lillooet (K7)","fire_centre":{"name":"Kamloops Fire Centre"}}}]}
```

Comment:
Is it possible to format this so it's a bit easier to read?

Reply:
Fixed